app/views/podcast_planner/_form_side.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="card">
+  <div class="card-header-primary">
+    <h5 class="card-title"><%= t(".header.preview") %></h5>
+  </div>
+  <%= turbo_frame_tag "preview", target: "_top" do %>
+    <% if @planner.dates.present? %>
+      <%= form_with(
+            url: podcast_planner_path(@podcast),
+            method: :post
+          ) do |form| %>
+          <div class="card-body">
+            <div class="col mb-2">
+              <h3 class="fw-bold"><%= t(".header.publish_time") %></h3>
+              <div class="form-floating">
+                <%= form.select :publish_time, time_options %>
+                <%= form.label :publish_time %>
+              </div>
+            </div>
+
+            <div class="col mb-2">
+              <h3 class="fw-bold"><%= t(".header.audio_segments") %></h3>
+              <div class="form-floating">
+                <%= form.number_field :segment_count, {min: 1, max: 10, step: 1, value: 2} %>
+                <%= form.label :segment_count %>
+              </div>
+            </div>
+
+            <div class="col">
+              <p><%= t(".header.number_of_drafts", count: @planner.dates.count) %></p>
+              <ul class="list-group list-group-numbered">
+                <% @planner.dates.each do |date| %>
+                  <li class="list-group-item"><%= l(date, format: :day_and_date) %></li>
+                  <%= form.hidden_field :generated_dates, multiple: true, value: date %>
```

Comment:
+10 ... and I think this will help when you get to the calendar UI.  Since the user should be able to select/unselect dates here _after_ generating the initial schedule.  (Maybe there are a few days they'll skip, or some additional bonus episodes to add).